ADApp/pluginTests/testingutilities.h

```diff
@@ -16,6 +16,7 @@
 #include <asynPortClient.h>
 
 void fillNDArrays(const std::vector<size_t>& dimensions, NDDataType_t dataType, std::vector<NDArray*>& arrays);
```

Comment:
Yes, it could be deleted if everyone is happy with requiring the use of NDArrayPool to create new NDArrays.